exercises/concept/chessboard/chessboard_test.go

```diff
@@ -32,74 +33,70 @@ func newChessboard() Chessboard {
 
 func TestCountInFile(t *testing.T) {
 	cb := newChessboard()
-	for _, test := range []struct {
-		in  string
-		out int
+	testCases := []struct {
+		in       string
+		expected int
 	}{
-		{"A", 3},
-		{"B", 1},
-		{"C", 1},
-		{"D", 0},
-		{"E", 2},
-		{"F", 0},
-		{"G", 1},
-		{"H", 7},
-		{"Z", 0},
-	} {
-		if out := CountInFile(cb, test.in); out != test.out {
-			t.Errorf(
-				"CountInFile(chessboard, '%v') returned %v while %v was expected\n",
-				test.in,
-				out,
-				test.out,
-			)
-		}
+		{in: "A", expected: 3},
+		{in: "B", expected: 1},
+		{in: "C", expected: 1},
+		{in: "D", expected: 0},
+		{in: "E", expected: 2},
+		{in: "F", expected: 0},
+		{in: "G", expected: 1},
+		{in: "H", expected: 7},
+		{in: "Z", expected: 0},
+	}
+	for _, test := range testCases {
+		t.Run(test.in, func(t *testing.T) {
+			if got := CountInFile(cb, test.in); got != test.expected {
```

Comment:
What do you think of adding a `description` field to each test case and use that as a test name? I think having a test named "A" or "1" can be confusing. With a description we could be more explicit, as in `"Count of pieces in file A"`, or  `"Count of pieces in rank 1"`